development/extensions/modification_to_extension.rst

```diff
@@ -672,18 +777,25 @@ And that's it. No file edits required for the template files aswell.
 Adding Events
 -------------
 
-You can also add events to your extensions php and template code. If you miss an event from the core, please post a topic into the `[3.x] Event Requests <https://area51.phpbb.com/phpBB/viewforum.php?f=111>`_-Forum and we will include it for the next release.
-We try to include a huge bunch of events by default, but surely we can not cover every place your MODs need to be covered.
+You can also add events to your extensions php and template code. If you miss an
+event from the core, please post a topic into the
+`[3.x] Event Requests <https://area51.phpbb.com/phpBB/viewforum.php?f=111>`_-Forum
```

Comment:
According to this [post](https://area51.phpbb.com/phpBB/viewtopic.php?p=279076#p279076) it now just needs to go in the tracker.